# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/faad2/Makefile.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.am~	2004-02-06 17:04:39.000000000 +0100
+++ ./Makefile.am	2005-09-13 02:20:23.000000000 +0200
@@ -2,7 +2,7 @@
 
 EXTRA_DIST = faad2.spec
 
-rpm:    Makefile
-       make dist
-       $(RPMBUILD) -ta $(PACKAGE)-$(VERSION).tar.gz
-       rm $(PACKAGE)-$(VERSION).tar.gz
+rpm: Makefile
+	make dist
+	$(RPMBUILD) -ta $(PACKAGE)-$(VERSION).tar.gz
+	rm $(PACKAGE)-$(VERSION).tar.gz
